models/users.js

```diff
@@ -306,6 +309,78 @@ const initializeUser = async (userId) => {
   return true;
 };
 
+/**
+ * Adds user data for verification by moderators
+ * @param userId {String} - RDS User Id
+ * @param discordId {Object} - Discord id of RDS user
```

Comment:
since this is a obj shall we call it discordId or simply discord

Reply:
Fixed